### PR TITLE
Portfolio:  Desktop Version

### PR DESCRIPTION
**Pull request by VictorOmarT**
Task completed in this PR:

- Make the Desktop version of the project
- Learn about @Media Queries
- Mobile-first design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,12 +10,18 @@
...
+        <a href="#" class="name-desk">Contact</a>
+        <img src="pictures/Icon-Mail.png" class="name-desk" alt="" />
+        <img src="pictures/Enabled.png" class="only-mob" alt="menu" />
+      </div>
     </header>
-    <section class="main-body">
+    <section class="main-body d-flex">
       <section>
         <h1>Hey There.<br />I’m Victor<br /></h1>
         <h3>I’am a Software Developer</h3>
```
Comment (Shaher-11) [inline index.html:27]: The text is overlapping here 
### Original design
right

### Your Project
wrong

File: index.html [hunk 2]
```diff
@@ -27,124 +33,257 @@ <h3>I’am a Software Developer</h3>
...
+          />
+        </div>
+        <div>
+          <h5>Multi-Post Stories</h5>
+          <p class="par-black">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <div>
```
Comment (Shaher-11) [inline index.html]: the section needs to be centered as shown below 
### Original design
right
### Your Project
wrong

File: index.html [hunk 3]
```diff
@@ -27,124 +33,257 @@ <h3>I’am a Software Developer</h3>
...
+        <div class="first-box name-desk">
+          <button class="last-b custom-font" type="submit">See project</button>
+        </div>
+        <div class="first-box1 name-desk">
+          <h2>Data Dashboard <br />Healthcare</h2>
+          <p>
+            A daily selection of privately personalized <br />
+            reads; no accounts or sign-ups required. <br />
+            has been the industry's standard
+          </p>
```
Comment (Shaher-11) [inline index.html:89]: The projects should be centered in columns as shown below
### Original design
right

### Your project
wrong

File: index.html [hunk 4]
```diff
@@ -27,124 +33,257 @@ <h3>I’am a Software Developer</h3>
...
+    <section class="laspart d-flex">
+      <div class="Goinlef">
+        <h1 class="en">About me</h1>
+        <p class="texf en">
+          Hello I’m a software developer! I can help you build a <br />
+          product , feature or website Look through some of my work <br />
+          and experience! If you like what you see and have a project <br />
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="butm en">Get My Resume</button>
```
Comment (Shaher-11) [inline index.html:187]: Add some space between the button and the paragraph 
### Original design
right

### Your Project
wrong

File: index.html [hunk 5]
```diff
@@ -27,124 +33,257 @@ <h3>I’am a Software Developer</h3>
...
+          <p class="endt">Skills</p>
+          <ul class="enlase">
+            <li><a href="#">Codekit</a></li>
+            <li><a href="#">Github</a></li>
+            <li><a href="#">Codepen</a></li>
+            <li><a href="#">Gitlab</a></li>
+            <li><a href="#">Terminal</a></li>
+          </ul>
         </div>
       </div>
```
Comment (Shaher-11) [inline index.html]: the section needs to be styled based on the given design below
### Original design
right

### Your Project
wrong

File: index.html [hunk 6]
```diff
@@ -27,124 +33,257 @@ <h3>I’am a Software Developer</h3>
...
+              />
+            </li>
+            <li>
+              <button class="last-b custom-font" type="submit">
+                Get in touch
+              </button>
+            </li>
+          </ul>
+        </div>
       </form>
```
Comment (Shaher-11) [inline index.html]: Reduce the width of the form and style the text as shown below
### Original Design
right

### Your Project
wrong

Comment (Shaher-11) [inline index.html:293]: Add the social links below the form
right

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @VictorOmarT,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html [hunk 7]
```diff
@@ -27,125 +33,273 @@ <h3>I’am a Software Developer</h3>
...
+              <img
+                class="resizer-handle"
+                src="./pictures/text_area/resizer_handle.png"
+                alt=""
+              />
+            </li>
+            <li>
+              <button class="last-b custom-font" type="submit">
+                Get in touch
+              </button>
```
Comment (ahmetbozaci) [inline index.html:289]: **Requirement**
- [x] Button position is not like in original design please fix it.
- [x] Also you can add placeholder in form box.

**Original:**
image

___
**Your design:**
image

File: index.html (hunk 4 above)
Comment (ahmetbozaci) [reply index.html:187]: Nice to fix last review.

### OPTIONAL
- [ ] Button  is not same like original. Also you can add button styling. Active,Hover,Pressed.

File: index.html [hunk 8]
```diff
@@ -10,12 +10,18 @@
...
-      </nav>
+      <div class="left-side-head">
+        <a href="#" class="name">Victor Toraño</a>
+      </div>
+      <div class="righ-side-head">
+        <a href="#" class="name-desk">Porfolio</a>
+        <a href="#" class="name-desk">About</a>
+        <a href="#" class="name-desk">Contact</a>
+        <img src="pictures/Icon-Mail.png" class="name-desk" alt="" />
+        <img src="pictures/Enabled.png" class="only-mob" alt="menu" />
```
Comment (ahmetbozaci) [inline index.html:21]: ### OPTIONAL
- [ ] Hamburger icon should appear only in mobile version. You can fix it.
- [ ]  when ı change screen width ı ı can scroll it horizontally. You can fix it too. Page should fit when you change screen width.
image

File: index.html [hunk 9]
```diff
@@ -27,125 +33,273 @@ <h3>I’am a Software Developer</h3>
...
+            reads; no accounts or sign-ups required. <br />
+            has been the industry's standard
+          </p>
+          <ul class="linksd">
+            <li class="transla"><img src="pictures/translu.png" alt="" /></li>
+            <li class="transla"><img src="pictures/translu1.png" alt="" /></li>
+            <li class="transla"><img src="pictures/translu2.png" alt="" /></li>
+          </ul>
+        </div>
+      </div>
```
Comment (ahmetbozaci) [inline index.html:176]: ### Optional
- [ ] I can't see project images. They are in there but not showing. Something wrong in your folder path. I am sure it is working  your side. But it is has to work every pc.  You can check with this [website](https://raw.githack.com)
- [ ] Also below 1024px your this section broken. You can fix it.
image

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️ 


## Hello @VictorOmarT ✋🏼  Nice done with working on last review.However there are still some issues you should fix. Please fix them then your project will be approved.





### ♻️ Request changes ♻️
 

- [ ] _**Check the comment about that  under the review.**_

### OPTIONAL SUGGESTIONS

#### These are optional but highly recommended.

- [ ] _**Check the comment about that  under the review.**_

- [ ] Your commit messages should always start  with capital letters and you should write an imperative tense. I see some of them are not like that. You can change them by using `git rebase`. Please watch this [video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) and check this [article](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
- [ ]  Instead of using pictures for icons you can use icons from [font-awesome](https://fontawesome.com/start). 



### Cheers and Happy coding! 💻 👏
cat on laptop



___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

File: index.html (hunk 7 above)
Comment (VictorOmarT) [reply index.html:289]: I don't know why but the form box always has blank spaces and looks like it doesn't have a placeholder but it does. :)

**Review by voscarmv — APPROVED** [review]
Hi @VictorOmarT , 

I've decided to approve this project because, officially, you have completed all the mandatory changes requested by @ahmetbozaci . Also, I've decided to save you the trouble of repeating the week, as most of the page seems fairly complete.

## HOWEVER ❗ 

There are just **too many** unmet optional changes in this project! And some of them are in fact mandatory according to the project requirements. Please take a step back and fix all the optional changes before merging!

I will make an exception this time, but it is **extremely important** that you fix these issues even after I've approved this project. In the future, your next Code Reviewer might not be as understanding as me and could ask you to repeat the week if they see that you have not met the necessary requirements.

Take this as a friendly warning!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (ahmetbozaci) [reply index.html:289]: - In design from box color is not visible so better adding placeholder.
- Also your brach names are not meaningful. You should write more descriptive name to your next projects.
[how to change branch name](https://linuxize.com/post/how-to-rename-local-and-remote-git-branch/)

**Pull request merged**